src/widgets/toggle-selector.ts

```diff
@@ -0,0 +1,35 @@
+import { Widget } from './widget';
+import { ElementArrayFinder } from '../wdio';
+
+export interface ToggleSelectorOption {
+    id: string;
+    caption: string;
+}
+
+export class ToggleSelector extends Widget {
+
```

Comment:
What about a method to select an option by text? 
If so, using this widget wouldn't require using the interface defined above where we are exposing an internal detail (id) of the toggle selector implementation.

Reply:
It was initially intended, but I deferred it (see description)